R/visualization_spatial.R

```diff
@@ -612,6 +727,8 @@ MapLabels.default <- function (
   pt_size = 1,
   pt_alpha = 1,
   pt_stroke = 0,
+  shape = "point",
+  spot_side,
```

Comment:
Should this have a default value?

Reply:
The value for `spot_side` is assigned when calling `MapLabels.default`, as we retrieve `spot_side` with `MapLabels.Seurat`. In `MapLabels.Seurat`, the default value is NULL. Should I do the same for the default value in `MapLabels.default`?

Reply:
Not so sure, but if you think it makes more sense to not have a default I would move up the argument to the top. It's good practice to put required arguments on the top and then parameters with defaults at the bottom.

Reply:
Argument has now been moved to the top of the function for the `.default` functions. In `.Seurat`, they stay with NULL.